creator/referral_tokens/migrations/0003_add_referral_token.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.24 on 2022-01-18 07:27
```

Comment:
This migration got autogenerated but probably shouldn't be included in this PR since its not related to the changes to add referral token to the event model. Seems like there is something wrong or some missing migration for referral token and organization but we can fix that in a later PR